### PR TITLE
db cleanup, after install complete

### PR DESCRIPTION
**Pull request by Bharath92**
https://github.com/Shippable/base/issues/962

### CODE REVIEW

File: scripts/cleanup.sh [hunk 1]
```diff
@@ -39,13 +42,51 @@ cleanup() {
...
+
 main() {
-  __process_marker "Cleaning up stale images"
-  
+  __process_marker "Cleaning up"
+
   if [ "$INSTALL_MODE" == "production" ]; then
-    cleanup
+    cleanup_stale_images
+    cleanup_db
```
Comment (shrivara) [inline scripts/cleanup.sh]: Lets clean up db first.

File: scripts/cleanup.sh [hunk 2]
```diff
@@ -1,6 +1,9 @@
 #!/bin/bash -e
+export CLEANUP_DIR="$MIGRATIONS_DIR/post_install"
```
Comment (shrivara) [inline scripts/cleanup.sh]: We should move this to base.sh and make it readonly.

File: scripts/cleanup.sh [hunk 3]
```diff
@@ -39,13 +42,51 @@ cleanup() {
...
+}
+
+clean_up_db_local() {
+  __process_msg "Cleaning up DB"
+
+  local db_username=$(cat $STATE_FILE | jq -r '.systemSettings.dbUsername')
+  local db_name="shipdb"
+
+  ##TODO: this should be the latest release file
+  ##TODO update the version in state after migration is run
```
Comment (shrivara) [inline scripts/cleanup.sh]: We should remove this TODO.

File: scripts/cleanup.sh (hunk 2 above)
Comment (shrivara) [inline scripts/cleanup.sh]: Also rename to POST_INSTALL_MIGRATIONS_DIR

File: migrations/post_install/v5.2.3-post_install.sql [hunk 4]
```diff
@@ -0,0 +1,7 @@
+do $$
```
Comment (ric03uec) [inline migrations/post_install/v5.2.3-post_install.sql]: @Bharath92 @shrivara not sure why we need a new file for this? can this not be done directly in the migrations.

**Pull request merged**